src/targets/browser/index.jsx

```diff
@@ -14,13 +15,15 @@ import manifest from '../../../manifest.webapp'
 import MuiCozyTheme from 'cozy-ui/transpiled/react/MuiCozyTheme'
 
 const RootApp = props => (
-  <CozyProvider client={props.client} store={props.store}>
-    <I18n lang={props.lang} polyglot={props.polyglot}>
-      <MuiCozyTheme>
-        <HotedApp />
-      </MuiCozyTheme>
-    </I18n>
-  </CozyProvider>
+  <Provider store={props.store}>
+    <CozyProvider client={props.client} store={props.store}>
```

Comment:
I had to put `store={props.store}` on `Provider` but also on `CozyProvider`. Am I missing something or is it ok ?

Reply:
It's ok.